js/apps/admin-ui/src/realm/add/NewRealmForm.tsx

```diff
@@ -29,14 +24,17 @@ export default function NewRealmForm() {
   const { refresh: refreshRealms } = useRealms();
   const { addAlert, addError } = useAlerts();
   const [realm, setRealm] = useState<RealmRepresentation>();
+  const [defaultValue, setDefaultValue] = useState(true);
```

Comment:
Why do we now need this additional state and effect? If you need to pass a default value you can do the following:

```
<DefaultSwitchControl defaultValue={true} />
```

Reply:
It doesn't work, I tried to modify DefaultSwitchControl component by adding defaultValue as a prop but it still complained that SwitchProps doesn't have defaultValue

Reply:
It should, we're also using `defaultValue` in [other places.](https://github.com/keycloak/keycloak/blob/4b697009d3336dbb702eecc83083020cc16b5de3/js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx#L160)

Reply:
This should be fixed now